node/service/src/chain_spec.rs

```diff
@@ -216,18 +216,26 @@ fn cord_local_genesis(
 	endowed_accounts: Option<Vec<AccountId>>,
 ) -> GenesisConfig {
 	let endowed_accounts: Vec<AccountId> = endowed_accounts.unwrap_or_else(testnet_accounts);
-
 	let num_endowed_accounts = endowed_accounts.len();
-	const ENDOWMENT: u128 = 50_000 * WAY;
+	const ENDOWMENT: u128 = 10_000 * WAY;
```

Comment:
Question: If the transaction payment is removed then why do we have/need `ENDOWMENT`?

Reply:
Balance is required now to enable the genesis accounts. We can remove this in a future PR.

Reply:
Good question. 

Yes, transaction fee is removed, but not the 'Balance' in the chain. As we 'start' a network, we still need to define a set of entities with starting Balances in the chain. Hence the ENDOWMENT is required.

Reply:
What exactly is the purpose of balance? How is it used internally on the chain?